<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Prediction + Analysis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Enter a project name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_prediction_project_name"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Choose a .pdb file or enter a PDB ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLineEdit" name="txt_prediction_load_reference"/>
          </item>
          <item>
           <widget class="QToolButton" name="btn_prediction_load_reference">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_prediction_chain_info">
          <property name="text">
           <string>Add chain information</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Chains in reference (input style: G,H)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_prediction_chain_ref"/>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Chains in models (input style: A,B)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_prediction_chain_model"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>195</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QPushButton" name="btn_prediction_cancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_prediction_start">
            <property name="text">
             <string>Predict with Colab Notebook</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progress_bar_prediction">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Home_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Analysis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Single Analysis</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Enter a project name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_analysis_project_name"/>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Choose a .pdb file or enter a PDB ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QLineEdit" name="txt_analysis_load_reference"/>
              </item>
              <item>
               <widget class="QToolButton" name="btn_analysis_load_reference">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btn_analysis_load_model">
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open model</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="txt_analysis_load_model">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_analysis_chain_info">
              <property name="text">
               <string>Add chain information</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Chains in reference (input style: G,H)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_analysis_chain_ref"/>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Chains in models (input style: A,B)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_analysis_chain_model"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>152</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Start structure prediction analysis</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_analysis_start">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="progress_bar_analysis">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Batch</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Enter a job name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_batch_job_name"/>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Choose a .pdb file or enter a PDB ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <item>
               <widget class="QLineEdit" name="txt_batch_load_reference"/>
              </item>
              <item>
               <widget class="QToolButton" name="btn_batch_load_reference">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QPushButton" name="btn_batch_load_model">
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open models</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="txt_batch_load_model">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_batch_chain_info">
              <property name="text">
               <string>Add chain information</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Chains in reference (input style: G,H)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_batch_chain_ref"/>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Chains in models (input style: A,B)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_batch_chain_model"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Start batch analysis</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_batch_start">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="progress_bar_batch">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="btn_view_distance_plot">
              <property name="text">
               <string>View distance plot</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_view_struct_alignment">
              <property name="text">
               <string>View structure alignment</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="btn_view_distance_histogram">
              <property name="text">
               <string>View distance histogram</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_view_interesting_regions">
              <property name="text">
               <string>View interesting regions</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="btn_view_distance_table">
              <property name="text">
               <string>View distance table</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5"/>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Image">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Representation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="box_representation"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Background Color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="box_bg_color"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Renderer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="box_renderer"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Ray-Trace-Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="box_ray_trace_mode"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Ray Texture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="box_ray_texture"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Ray-Tracing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_ray_tracing">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Transparent Background</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_transparent_bg">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>238</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="btn_save_scene">
            <property name="text">
             <string>Save Scene</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_update_scene">
            <property name="text">
             <string>Update Scene</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_preview_image">
            <property name="text">
             <string>Preview Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_save_image">
            <property name="text">
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_file_open"/>
    <addaction name="action_file_save_as"/>
    <addaction name="action_file_save"/>
    <addaction name="separator"/>
    <addaction name="action_file_restore_settings"/>
    <addaction name="action_file_quit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_settings_edit_all"/>
    <addaction name="action_settings_change_workspace"/>
    <addaction name="action_settings_model_w_off_colab_notebook"/>
   </widget>
   <widget class="QMenu" name="menuWizard">
    <property name="title">
     <string>&amp;Wizard</string>
    </property>
    <addaction name="action_wizard_prepare_model_pdbs"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_help_docs"/>
    <addaction name="action_help_docs_pdf"/>
    <addaction name="separator"/>
    <addaction name="action_help_about"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>&amp;Display</string>
    </property>
    <addaction name="action_display_current_workspace"/>
    <addaction name="action_display_project_path"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuWizard"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="action_file_save_as">
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
  <action name="action_file_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_wizard_prepare_model_pdbs">
   <property name="text">
    <string>Prepare Models</string>
   </property>
  </action>
  <action name="actSettingsPdbPreparation">
   <property name="text">
    <string>pdb Preparation</string>
   </property>
  </action>
  <action name="action_file_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_help_docs">
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
  </action>
  <action name="action_help_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_settings_edit_all">
   <property name="text">
    <string>&amp;Edit all</string>
   </property>
  </action>
  <action name="action_settings_change_workspace">
   <property name="text">
    <string>Change Workspace</string>
   </property>
  </action>
  <action name="action_file_restore_settings">
   <property name="text">
    <string>Restore Settings</string>
   </property>
  </action>
  <action name="action_settings_model_w_off_colab_notebook">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Modeling with &quot;official&quot; colab notebook</string>
   </property>
  </action>
  <action name="action_display_current_workspace">
   <property name="text">
    <string>Current &amp;Workspace Path</string>
   </property>
  </action>
  <action name="action_display_project_path">
   <property name="text">
    <string>Current &amp;Project Path</string>
   </property>
  </action>
  <action name="action_help_docs_pdf">
   <property name="text">
    <string>Documentation P&amp;DF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
